.pre-commit-config.yaml

```diff
@@ -202,7 +202,7 @@ repos:
     hooks:
     -   id: no-string-hints
 -   repo: https://github.com/MarcoGorelli/abs-imports
-    rev: v0.1.2
+    rev: v0.1.3
     hooks:
     -   id: abs-imports
-        files: ^pandas/
+        args: [--application-directories, '.,./asv_bench']
```

Comment:
what are the commas?

Reply:
application directories, comma-separated. So, the first one is `.` (which contains `pandas`), and the second one is `./asv_bench` (which contains `benchmarks`)

Reply:
maybe its the quotes as you have a comma *inside* the 2nd one, this is really odd. but ok

Reply:
yes, cause otherwise yaml doesn't parse it correctly. maybe a semi-colon would be a better separator